### ISSUE TITLE
Manager and agent instances are not created

### ISSUE THREAD

**Issue by HarryEMartland** [fluxninja/aperture#1319]
**Describe the bug**
Manager and Agent instances are not created

**Steps to reproduce**
Using EKS, apply helm charts

**What did you expect to see?**
Manager and agent instances are created

**What did you see instead?**
Manager and agent instances are not created and logs below

**What version did you use?**
aperture-agent-v0.22.0
aperture-controller-v0.22.0

**What config did you use?**
Config: (e.g. the yaml config file)
As json as using CDK
```
controller: {
   config: {
      plugins: {disabled_plugins: ["aperture-plugin-fluxninja"]}
   }
}
```

**Environment**
Kubernetes version 1.23
EKS

**Additional context**
Logs from managers

***Controller Manager***
`E0207 17:04:57.004138       1 leaderelection.go:330] error retrieving resource lock default/a4362587-controller.fluxninja.com: leases.coordination.k8s.io "a4362587-controller.fluxninja.com" is forbidden: User "system:serviceaccount:default:aperture-controller" cannot get resource "leases" in API group "coordination.k8s.io" in the namespace "default"`

***Agent Manager***
`E0207 17:05:40.069559       1 leaderelection.go:367] Failed to update lock: leases.coordination.k8s.io "a4362587-agent.fluxninja.com" is forbidden: User "system:serviceaccount:default:aperture-agent" cannot update resource "leases" in API group "coordination.k8s.io" in the namespace "default"`

**Comment by hdkshingala** [comment]
@HarryEMartland Could you please confirm the helm command you have used for above installation? 

If it is using `aperture-controller` and `aperture-agent` as names with `helm install` command, could you please try with some other name? An issue was observed with these names which will be fixed as part of #1318

**Comment by hdkshingala** [comment]
Thank you for confirming @HarryEMartland

For CKD, based on https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-eks.HelmChart.html, it seems like `release` field in the example you shared is the field which you can try changing with some other value for fixing this issue.

```
let agentHelm = cluster.addHelmChart('aperture-agent', {
    chart: 'aperture-agent',
    repository: 'https://fluxninja.github.io/aperture/',
    namespace,
    timeout: Duration.minutes(1),
    values: {
        agent: {
            create: true,
            config: {
                etcd: {endpoints: ["http://aperture-etcd.<removed>"]},
                prometheus: {address: "http://aperture-prometheus.<removed>"},
                plugins: {disabled_plugins: ["aperture-plugin-fluxninja"]}
            }
        }
    },
    release: "agent"
});
```
